### ISSUE TITLE
docker build command doesn't run

### ISSUE THREAD

**Issue by ELchem** [deepdoctection/deepdoctection#129]
**Bug** 💥
running `docker build -t dd:<your_tag> -f docker/pytorch-cpu-jupyter` fails.
removing -f flag yields the following error:
```
 ERROR: Directory '.[source-pt]' is not installable. Neither 'setup.py' nor 'pyproject.toml' found.
```

**Expected behavior**  🧮 
docker container starts as expected.

**Screenshots** 🖼
![image](https://user-images.githubusercontent.com/54755533/225193265-5c76f0b0-83e6-4a24-8597-6968b78a7fa9.png)

**Desktop (please complete the following information, if any other than the one in the install requirements):**
 - OS: windows11
 - docker: Docker version 20.10.14, build a224086
 - python: 3.11.2

**Comment by JaMe76** [comment]
Thanks. 

Can you try instead

```
docker build -t dd:<your-tag> -f docker/pytorch-cpu-jupyter/Dockerfile .
``` 

This works on my local machine (Ubuntu 20.04) and creates an image that can be run with the following commands.

**Comment by JaMe76** [comment]
Doc fixed with 5fb23557f07d30c241a9b1fda7f42cb0f59f05cb

**Comment by ELchem** [comment]
Thanks that fixed the issue!
